app/src/components/UI/Pages/Vendor.tsx

```diff
@@ -118,6 +119,12 @@ export function Vendor(): React.ReactElement {
             <h1 className={styles.name}>{vendor?.Name}</h1>
             <VendorStar vendorID={vendorID} />
           </Grid.Row>
+          <TwitterShareButton
+            url="http://localhost:3000/streetfoodlove/vendors/e72ac985-3d7e-47eb-9f0c-f8e52621a708"
+            title="Check out this Vendor!"
+          >
+            <TwitterIcon size={32} round={true} />
+          </TwitterShareButton>
```

Comment:
This definitely looks like it links to the Twitter page for this vendor, not to share the vendor. Could you add a label or something that indicates that it shares a link to this page? Also, maybe it should be lower on the page because it is not as important as the vendor information.

Reply:
I agree that its not as important, but I think where it is right now is the best, because it is not related to reviews or the vendor information cards. Making the icon smaller like it is would make it less important then the other elements already on the page, I believe.